spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -103,6 +103,14 @@ public Template getTemplate() {
 		return this.template;
 	}
 
+	/**
+	 * Whether transactions are enabled for either the template or the listener.
+	 * @return whether transactions are enabled for either the template or the listener
+	 */
+	public boolean isTransactionEnabled() {
```

Comment:
This is a convenience that is currently used by the `PulsarPropertiesMapper` to decide whether or not to enable the Pulsar client w/ transaction support. We do that if at least one of `spring.pulsar.template.transaction.enabled` or `spring.pulsar.listener.transaction.enabled` is `true`.  I was not sure if this is the best place to add that logic but it seemed better here than in the properties mapper.

Reply:
Done

Reply:
As this was only used by the mapper, I've moved it there. Even with it being package-private, I think it reads better if the code's closer to where it's used.